### PR TITLE
frontend: fix actions btns spacing when collapsed

### PR DESCRIPTION
**Pull request by shonsirsha**
Before:
2e

After:
ewda

asdsa

### CODE REVIEW

**Review by thisconnect — CHANGES_REQUESTED** [review]
I fount a tiny somewhat related bug:

it seems that the WalletConnect button text disappears but the icon is not centered.

Screen Shot 2024-01-02 at 19 41 37

Since you are here could you look into that as well?

File: frontends/web/src/routes/account/account.module.css [hunk 1]
```diff
@@ -77,14 +79,15 @@
...
         margin-bottom: var(--space-default);
         max-width: 91%;
-        transform: none;
-        width: 91%;
         margin-left: auto;
         margin-right: auto;
-        flex-wrap: wrap;
+        padding-bottom: 0;
+        transform: none;
+        width: 91%;
```
[the remark this answers is not in the document]
Comment (shonsirsha) [reply frontends/web/src/routes/account/account.module.css:90]: hehe, wouldn't know either. My best bet would be due to eyeballing the size of the buttons. But 100% looks fine as well. I'll remove the max-width too as that seems to be not useful as well as the `flex` shorthand property.

**Comment by shonsirsha** [comment]
PTAL @thisconnect , fixed the bug you found by always the centreing the items in the button.

**Review by thisconnect — APPROVED** [review]
tested LGTM

**Pull request merged**